Document/0x05a-Platform-Overview.md

```diff
@@ -594,8 +594,8 @@ The <code>zipalign</code> tool should always be used to align an APK file before
 
 #### Publishing Process
 
-The Android ecosystem is open, and, as such, it is possible to distribute apps from anywhere (your own site, any store, ...). However, Google Play is the more famous, trusted and popular store and is provided by Google itself.
+The Android ecosystem is open, and, as such, it is possible to distribute apps from anywhere (your own site, any store, ...). However, Google Play is the more famous, trusted and popular store and is provided by Google itself. Amazon Appstore is the default, trusted store on Kindle devices. A user must explicitly allow 3rd party apps to be installed on their device from unknown source.

```

Comment:
> A user must explicitly allow 3rd party apps to be installed on their device from unknown source.

This sounds like a user must always do this, but I think he only needs to do this if he wants to use other sources for apps.

Also, please write out "third party".